model/task/db.go

```diff
@@ -616,6 +616,74 @@ func GetRecentTasks(period time.Duration) ([]Task, error) {
 	return tasks, nil
 }
 
+type StatsList struct {
```

Comment:
It feels unnecessary to have both []StatsList and ResultCountList. Is there a way to have the aggregation return results in the latter format?

Reply:
I moved things around and now the documents returned by the aggregation need to be reshaped into the ResultCountList.